File: AndroidProject_team4_plantalk_spring_server/src/main/java/com/and4/travel_server/model/Diary.java
package com.and4.travel_server.model;

import lombok.Data;

@Data
public class Diary {

	private int dno;
	private String title;
	private String content;
	private String date;
	private String on_off;
	private int hitcount;
	private int good;
	private int trip_id;
	private String image_uri;
}

File: AndroidProject_team4_plantalk_app/app/src/main/java/com/android4/travel/model/Diary.kt
package com.android4.travel.model

import java.util.*

data class Diary(
    var dno:Int,
    var title:String,
    var content:String,
    var date: String,
    var on_off: String,
    var hitcount:Int,
    var good:Int,
    var trip_id:String,
    var image_url: String
)
